.gitpod.Dockerfile

```
@@ -2,18 +2,19 @@ FROM gitpod/workspace-full:latest
 
 # Magento Config
 ENV INSTALL_MAGENTO YES
-ENV MAGENTO_VERSION 2.4.4
+ENV MAGENTO_VERSION 2.4.7-p1
```

```suggestion
ENV MAGENTO_VERSION 2.4.7-p3
```

hey @johnhughes1984, would it be possible to update this to 2.4.7-p3?